.github/workflows/create-release-candidate.yml

```
@@ -1,12 +1,13 @@
-name: Create Release Candidate (6.x)
+name: Create Release Candidate
```

```suggestion
name: Create Release
```

and also rename this file to `create-release.yml`?

---

Yep! Why not